### PR TITLE
Mobile teams & chat - more visual tweaks

### PR DESCRIPTION
**Pull request by cecileboucheron**

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits 5667da45a1b0...5d3ee764e4ee introduce visual changes on linux. <details><summary>:mag_right: 13 changed</summary>

 * Changed: **Popup_Dialog-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Popup_Dialog-Normal-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Popup_Dialog-Normal-5d3ee764e4ee.png)  
   Popup_Dialog-Normal comparison
 * Changed: **Notifications-Unsub_All** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Notifications-Unsub_All-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Notifications-Unsub_All-5d3ee764e4ee.png)  
   Notifications-Unsub_All comparison
 * Changed: **Notifications-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Notifications-Normal-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Notifications-Normal-5d3ee764e4ee.png)  
   Notifications-Normal comparison
 * Changed: **Chat_Inbox-You_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-You_Rekey_Selected-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-You_Rekey_Selected-5d3ee764e4ee.png)  
   Chat_Inbox-You_Rekey_Selected comparison
 * Changed: **Chat_Inbox-You_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-You_Rekey-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-You_Rekey-5d3ee764e4ee.png)  
   Chat_Inbox-You_Rekey comparison
 * Changed: **Chat_Inbox-Selected_Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Selected_Normal-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Selected_Normal-5d3ee764e4ee.png)  
   Chat_Inbox-Selected_Normal comparison
 * Changed: **Chat_Inbox-Selected_Muted** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Selected_Muted-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Selected_Muted-5d3ee764e4ee.png)  
   Chat_Inbox-Selected_Muted comparison
 * Changed: **Chat_Inbox-Part_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Part_Rekey_Selected-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Part_Rekey_Selected-5d3ee764e4ee.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Part_Rekey_Selected.png)
 * Changed: **Chat_Inbox-Part_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Part_Rekey-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Part_Rekey-5d3ee764e4ee.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Part_Rekey.png)
 * Changed: **Chat_Inbox-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Normal-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Normal-5d3ee764e4ee.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Normal.png)
 * Changed: **Chat_Inbox-Long_Top** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Long_Top-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Long_Top-5d3ee764e4ee.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Long_Top.png)
 * Changed: **Chat_Inbox-Long_Bottom** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Long_Bottom-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Long_Bottom-5d3ee764e4ee.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Long_Bottom.png)
 * Changed: **Chat_Inbox-Empty** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Empty-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Empty-5d3ee764e4ee.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505257513981-5667da45a1b0-5d3ee764e4ee-linux/Chat_Inbox-Empty.png)
</summary>

File: shared/chat/inbox/row/chat-filter-row.js [hunk 1]
```diff
@@ -120,16 +110,48 @@ class _ChatFilterRow extends Component<Props, State> {
...
+  ...globalStyles.flexBoxRow,
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  minHeight: 48,
+  paddingLeft: globalMargins.small,
+  paddingRight: globalMargins.small,
+  position: 'relative',
+}
+
+const styleContainerMobile = {
```
Comment (akalin-keybase) [inline shared/chat/inbox/row/chat-filter-row.js]: this looks like it doesn't change `styleContainer`. Did you mean it to be different? If not, remove?

Comment (cecileboucheron) [reply shared/chat/inbox/row/chat-filter-row.js]: Ah yes! Removed.

**Comment by keybase-ci-visdiff** [comment]
The commits 5667da45a1b0...1804d0286f15 introduce visual changes on linux. <details><summary>:mag_right: 13 changed</summary>

 * Changed: **Popup_Dialog-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Popup_Dialog-Normal-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Popup_Dialog-Normal-1804d0286f15.png)  
   Popup_Dialog-Normal comparison
 * Changed: **Notifications-Unsub_All** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Notifications-Unsub_All-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Notifications-Unsub_All-1804d0286f15.png)  
   Notifications-Unsub_All comparison
 * Changed: **Notifications-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Notifications-Normal-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Notifications-Normal-1804d0286f15.png)  
   Notifications-Normal comparison
 * Changed: **Chat_Inbox-You_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-You_Rekey_Selected-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-You_Rekey_Selected-1804d0286f15.png)  
   Chat_Inbox-You_Rekey_Selected comparison
 * Changed: **Chat_Inbox-You_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-You_Rekey-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-You_Rekey-1804d0286f15.png)  
   Chat_Inbox-You_Rekey comparison
 * Changed: **Chat_Inbox-Selected_Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Selected_Normal-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Selected_Normal-1804d0286f15.png)  
   Chat_Inbox-Selected_Normal comparison
 * Changed: **Chat_Inbox-Selected_Muted** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Selected_Muted-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Selected_Muted-1804d0286f15.png)  
   Chat_Inbox-Selected_Muted comparison
 * Changed: **Chat_Inbox-Part_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Part_Rekey_Selected-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Part_Rekey_Selected-1804d0286f15.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Part_Rekey_Selected.png)
 * Changed: **Chat_Inbox-Part_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Part_Rekey-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Part_Rekey-1804d0286f15.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Part_Rekey.png)
 * Changed: **Chat_Inbox-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Normal-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Normal-1804d0286f15.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Normal.png)
 * Changed: **Chat_Inbox-Long_Top** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Long_Top-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Long_Top-1804d0286f15.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Long_Top.png)
 * Changed: **Chat_Inbox-Long_Bottom** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Long_Bottom-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Long_Bottom-1804d0286f15.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Long_Bottom.png)
 * Changed: **Chat_Inbox-Empty** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Empty-5667da45a1b0.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Empty-1804d0286f15.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505264861278-5667da45a1b0-1804d0286f15-linux/Chat_Inbox-Empty.png)
</summary>

**Comment by cjb** [comment]
Pushed a merge with conflict fixed.

**Comment by keybase-ci-visdiff** [comment]
The commits 08a22e5ad44d...044edd3139a6 introduce visual changes on linux. <details><summary>:mag_right: 13 changed</summary>

 * Changed: **Popup_Dialog-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Popup_Dialog-Normal-08a22e5ad44d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Popup_Dialog-Normal-044edd3139a6.png)  
   Popup_Dialog-Normal comparison
 * Changed: **Notifications-Unsub_All** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Notifications-Unsub_All-08a22e5ad44d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Notifications-Unsub_All-044edd3139a6.png)  
   Notifications-Unsub_All comparison
 * Changed: **Notifications-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Notifications-Normal-08a22e5ad44d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Notifications-Normal-044edd3139a6.png)  
   Notifications-Normal comparison
 * Changed: **Chat_Inbox-You_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-You_Rekey_Selected-08a22e5ad44d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-You_Rekey_Selected-044edd3139a6.png)  
   Chat_Inbox-You_Rekey_Selected comparison
 * Changed: **Chat_Inbox-You_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-You_Rekey-08a22e5ad44d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-You_Rekey-044edd3139a6.png)  
   Chat_Inbox-You_Rekey comparison
 * Changed: **Chat_Inbox-Selected_Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Selected_Normal-08a22e5ad44d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Selected_Normal-044edd3139a6.png)  
   Chat_Inbox-Selected_Normal comparison
 * Changed: **Chat_Inbox-Selected_Muted** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Selected_Muted-08a22e5ad44d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Selected_Muted-044edd3139a6.png)  
   Chat_Inbox-Selected_Muted comparison
 * Changed: **Chat_Inbox-Part_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Part_Rekey_Selected-08a22e5ad44d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Part_Rekey_Selected-044edd3139a6.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Part_Rekey_Selected.png)
 * Changed: **Chat_Inbox-Part_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Part_Rekey-08a22e5ad44d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Part_Rekey-044edd3139a6.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Part_Rekey.png)
 * Changed: **Chat_Inbox-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Normal-08a22e5ad44d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Normal-044edd3139a6.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Normal.png)
 * Changed: **Chat_Inbox-Long_Top** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Long_Top-08a22e5ad44d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Long_Top-044edd3139a6.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Long_Top.png)
 * Changed: **Chat_Inbox-Long_Bottom** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Long_Bottom-08a22e5ad44d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Long_Bottom-044edd3139a6.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Long_Bottom.png)
 * Changed: **Chat_Inbox-Empty** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Empty-08a22e5ad44d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Empty-044edd3139a6.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1505274309461-08a22e5ad44d-044edd3139a6-linux/Chat_Inbox-Empty.png)
</summary>

**Pull request merged**